src/coord/src/coord.rs

```diff
@@ -4753,19 +4751,19 @@ pub mod fast_path_peek {
                         })?;
                     // We should only get excited if we can track down an index for `id`.
                     // If `keys` is non-empty, that means we think one exists.
-                    for (index_id, (desc, _typ)) in dataflow_plan.index_imports.iter() {
+                    for idx in dataflow_plan.index_imports.values() {
```

Comment:
Maybe a nit, but things make more sense to me when the enumeration is `(key, val)`. Does it screw things up here to have `index_id` outside of the `IndexDesc`? This is how `SourceDesc` works, and it seems like having them be similar could reduce the cognitive overhead.